python/taichi/lang/kernel_impl.py

```diff
@@ -188,7 +191,12 @@ def _process_args(self, args, kwargs):
 
     for i, arg in enumerate(ret):
         if arg is inspect.Parameter.empty:
-            raise TaichiSyntaxError(f"Parameter '{self.arguments[i].name}' missing.")
+            if self.arguments[i].annotation is inspect._empty:
+                raise TaichiSyntaxError(f"Parameter `{self.arguments[i].name}` missing.")
```

Comment:
This line caused `test_argument.py::test_function_keyword_args_missing` to fail.